Common++/src/TablePrinter.cpp

```diff
@@ -88,13 +89,7 @@ void TablePrinter::printSeparator()
 		return;
 	}
 
-	int totalLen = 0;
-	for (std::vector<int>::iterator iter = m_ColumnWidths.begin(); iter != m_ColumnWidths.end(); iter++)
-	{
-		totalLen += 2 + (*iter) + 1;
-	}
-
-	totalLen++;
+	int totalLen = std::accumulate(m_ColumnWidths.begin(), m_ColumnWidths.end(), m_ColumnWidths.size() * 3) + 1;
```

Comment:
nit: maybe use `auto` for `totalLen` and for the `for` loop later?

Reply:
@seladb What about this e38dfa1 ? I also removed `for` loop and use string constructor